.circleci/config.yml

```diff
@@ -714,6 +714,95 @@ jobs:
             - "/root/.cache/go-build"
       - notify-failures-on-develop
 
+  contracts-bedrock-tests-upgrade:
```

Comment:
I'm sure this could be done more elegantly, but it just copied and pasted from the normal `contracts-bedrock-tests` job for now to get something running in CI